packages/cohort/src/pages/CohortDetails.js

```diff
@@ -241,20 +249,24 @@ const CohortDetails = ({ footerLinks, setAlert, appName }) => {
       .toString()
       .padStart(2, "0")}`;
 
+    // Get end time safely
+    const selfAttendanceEnd =
+      cohortDetails?.params?.self_attendance_end ?? "23:59";
```

Comment:
@vaibhavsTekdi 
      cohortDetails?.params?.self_attendance_end ?? "23:59";
      this code, Is this correct?

Reply:
Yes, in case if the 'self_attendance_end' time is not added or forgot to add or somehow it is blanked that end time will be 23.59 that is for whole day.